draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1984,6 +1984,8 @@ data. The original JSON data is then used by the application. See
 
    -19
 
+    * Update the ABNF to something that is cleaner, less ambiguous, and more idiomatic
```

Comment:
Well, *[foo] is not ambiguous in the strict sense, it is just laying a trap by nesting repetition indicators.

Reply:
fair, I was just using the langue AI gave me

Reply:
```suggestion
    * Update the ABNF to something that is cleaner and more idiomatic
```